File: 04_조준장/10430.py
import sys

A, B, C = map(int, sys.stdin.readline().split())

# 문제의 조건에 맞게 출력
print((A + B) % C)
print(((A % C) + (B % C)) % C)
print((A * B) % C,)
print(((A % C) * (B % C)) % C)

File: 18_정지윤/10430.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    private void solve() {
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String[] str = br.readLine().split(" ");

            int A = Integer.parseInt(str[0]);
            int B = Integer.parseInt(str[1]);
            int C = Integer.parseInt(str[2]);

            System.out.println((A+B)%C);
            System.out.println(((A%C) + (B%C))%C);
            System.out.println((A*B)%C);
            System.out.println(((A%C) * (B%C))%C);

        }catch (IOException e) {
            e.printStackTrace();
        }
    }
    public static void main(String[] args){
        new Main().solve();
    }
}